### ISSUE TITLE
Cannot find namespace '$rdf'

### ISSUE THREAD

**Issue by tiagosomda** [Inrupt-inc/generator-solid-angular#13]
I went through the steps described on the readme.md and when I ran 'ng serve'

I was getting the following errors:
```
ERROR in src/app/services/rdf.service.ts(33,12): error TS2503: Cannot find namespace '$rdf'.
src/app/services/rdf.service.ts(41,18): error TS2503: Cannot find namespace '$rdf'.
src/app/services/rdf.service.ts(323,58): error TS2503: Cannot find namespace '$rdf'.
```
I checked the file src/app/services/rdf.service.ts
and the following line was commented out:
`import * as $rdf from 'rdflib'`

I uncommented it and the project built and I was able to load the page.

@james-martin-jd - is there a reason why that line needed to commented out?
I saw that you recently commented it out to fix an "issue where rdflib was throwing errors"
https://github.com/Inrupt-inc/generator-solid-angular/commit/1c91ae705d3b37f6567d929273154583d7012e4a#diff-dca975bf5ad72321f0990d8416fddab0

**Comment by tiagosomda** [comment]
Created a pull request (https://github.com/Inrupt-inc/generator-solid-angular/pull/14) if you think we should uncomment it

**Comment by james-martin-jd** [comment]
That's interesting, I was getting the opposite issue where I was unable to import rdflib using this method. This was changed in a PR, where we were just declaring the variable and including it in the index.html, so I reverted back to that method to prevent the errors, which worked locally. I'll investigate.

**Comment by tiagosomda** [comment]
Do you need some info from my machine/environment to help debug that?

**Comment by james-martin-jd** [comment]
Thanks for the offer! I'm checking into it right now - I doubt it's specific to an OS at the moment. I wonder if it's something locally on my dev machine that's configured weird. Like an extra package declaration or something.

**Comment by james-martin-jd** [comment]
To follow up on this, I opened an issue on rdflib github to fix imports. When I use the import instead of <script> and declare, I get errors related to a missing package dependency, so I think this is something that needs to be fixed. Once that gets addressed I'll change everything to import, as that's the preferred way of adding files.

**Comment by james-martin-jd** [comment]
Fixed in 0.1.6 release
